pkg/operator/ceph/config/defaults.go

```diff
@@ -17,10 +17,36 @@ limitations under the License.
 // Package config provides default configurations which Rook will set in Ceph clusters.
 package config
 
+import (
+	"bytes"
+	"fmt"
+)
+
 // DefaultFlags returns the default configuration flags Rook will set on the command line for all
 // calls to Ceph daemons and tools. Values specified here will not be able to be overridden using
 // the mon's central KV store, and that is (and should be) by intent.
 func DefaultFlags(fsid, mountedKeyringPath string) []string {
+	return defaultFlagConfigs(fsid, mountedKeyringPath).GlobalFlags()
+}
+
+// DefaultFlagsAsConfigFile is the same as DefaultFlags except that it returns the ceph.conf file
+// text which corresponds to the flags. In some versions of Ceph and for some Daemons, the default
+// flags are not accepted in flag form.
+func DefaultFlagsAsConfigFile(fsid, mountedKeyringPath string) (string, error) {
+	c := defaultFlagConfigs(fsid, mountedKeyringPath)
+	c.Section("global").Set("mon-initial-members", "a b c") // TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST TEST
```

Comment:
need to remove this line?

Reply:
This is still needed for the `mon --mkfs ...`, so to remove this line, that would need to be added in the mon spec. It's a todo item, but I have been planning it for a future, small PR.

Reply:
Oh oh oh oh oh ! Yes I really need to delete the `TEST TEST TEST TEST TEST` line. :scream: